atd-vzd/migrations/migration_atd_txdot_crashes_2023_08_03--1127.sql

```diff
@@ -0,0 +1,15 @@
+-- Create a stored generated column that finds the location id of a crash 
+-- using its geographic position and keeps it updated.
+ALTER TABLE atd_txdot_crashes
+ADD COLUMN generated_location_id text GENERATED ALWAYS AS (get_cr3_location_id(position, crash_id, rpt_road_part_id, rpt_hwy_num)) STORED;
+
+-- TODO drop DB views that use location_id (replace later with generated location_id)
+
+-- Drop current atd_txdot_crashes location_id column
+ALTER TABLE atd_txdot_crashes DROP COLUMN location_id;
+
+-- Rename generated_location_id to location_id to replace it
+ALTER TABLE atd_txdot_crashes 
+RENAME COLUMN generated_location_id TO location_id;
+
+-- TODO add DB views that use location_id
```

Comment:
following pattern from https://github.com/cityofaustin/atd-vz-data/pull/1269 for steps to retire the old column and rename the new generated column

Reply:
cool! fyi, I'm going to fill in these TODOs in my PR on Monday and then request review. I felt way to scattered today to untangle the dependencies that block dropping the current non-CR3 location column.

Reply:
@mddilley okay sounds good, i figured id just post the rest of my code for review now but ill also look into what it will take to drop the old column :-)